main.go

```diff
@@ -76,12 +77,26 @@ func BubbleSort(items [][]string) {
 			}
 		}
 	}
-
 }
-func LoadData() [][]string {
-	r, _ := os.Open("data/commits.csv")
+
+func LoadData() (records [][]string, err error) {
+	r, err1 := os.Open("data/commits.csv")
```

Comment:
No need to name these err1 and err2, you can just re-assign err.

Reply:
k

Reply:
is there variable scope issues when the function parameter specifies return variables of `records [][]string, err error` and I also use the same variable names within the body of function?